deploy/operator.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       containers:
         - name: operand-deployment-lifecycle-manager
           # Replace this with the built image name
-          image: quay.io/opencloudio/odlm:latest
+          image: quay.io/opencloudio/odlm:1.1.0
```

Comment:
according to cicd process, seems we should always push `latest` and the hash tag.

CICD will publish the final released tag.

Reply:
Yea,  I will remove the release version image push in the script.
Should I change the `1.1.0` back to `latest` here? I can move this change to my RH-certify branch to make sure the release version is `1.1.0`.

Reply:
Adam mentions
```
In addition to this, now that the release tags exist on quay.io, we need to update all the CSVs for all the operators in quay so that they point to the release tagged version of the operator instead of latest.
```
Then we don't need to change this line.